### PR TITLE
exercises: IntroduceLetter

### LINKED ISSUE #13: Implement Intro for Letter

### PR DESCRIPTION
**Pull request by metzgaria**
Closes #13

### CODE REVIEW

**Comment by inyono** [comment]
Note: ignore the commits, I will squash and rename them anyway

**Comment by inyono** [comment]
@Knorrke did you find time to review this?

File: src/components/exercises/IntroduceLetter.stories.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { storiesOf } from '@kadira/react-native-storybook';
+
+import IntroduceLetter from './IntroduceLetter';
+
+storiesOf('components/IntroduceLetter', module).add('default', () => (
+  <IntroduceLetter
+    images={[
+      require('../assets/images/ananas.jpg'),
```
Comment (Knorrke) [inline src/components/exercises/IntroduceLetter.stories.js]: wrong path. Please change this to `require('../../assets/images/ananas.jpg')`

File: src/components/exercises/IntroduceLetter.stories.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      require('../assets/images/apfel.jpg'),
```
Comment (Knorrke) [inline src/components/exercises/IntroduceLetter.stories.js]: wrong path. Please change this to `require('../../assets/images/apfel.jpg')`

File: src/components/exercises/IntroduceLetter.stories.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      require('../assets/images/affe.jpg')
```
Comment (Knorrke) [inline src/components/exercises/IntroduceLetter.stories.js]: wrong path. Please change this to `require('../../assets/images/affe.jpg')`

**Comment by Knorrke** [comment]
Found out how to add changes to the branch myself 😄 Should be ok to merge now @inyono 

Thanks @metzgaria for the nice work! :)

**Pull request merged**